CI fails due to:
```log
-- ITKIOMeshOBJ: Creating itkOBJMeshIO submodule.
CMake Error at CMake/ITKModuleTest.cmake:135 (message):
  Duplicate test name 'itkOBJMeshIOPythonTest' (already added in
  /home/vsts/work/1/s/Modules/Core/Mesh/wrapping/test/CMakeLists.txt).
Call Stack (most recent call first):
  CMake/ITKModuleTest.cmake:239 (itk_add_test)
  CMake/ITKModuleTest.cmake:309 (itk_python_add_test)
  Modules/IO/MeshOBJ/wrapping/test/CMakeLists.txt:2 (itk_python_expression_add_test)
```

--- a/Modules/Core/Mesh/wrapping/test/CMakeLists.txt
+++ b/Modules/Core/Mesh/wrapping/test/CMakeLists.txt
@@ -24,14 +24,6 @@ if(ITK_WRAP_PYTHON)
     EXPRESSION "instance = itk.MeshFileWriter.New()"
   )
   itk_python_expression_add_test(
-    NAME itkOBJMeshIOPythonTest
-    EXPRESSION "instance = itk.OBJMeshIO.New()"
-  )
-  itk_python_expression_add_test(
-    NAME itkOFFMeshIOPythonTest
-    EXPRESSION "instance = itk.OFFMeshIO.New()"
-  )
-  itk_python_expression_add_test(
     NAME itkVTKPolyDataMeshIOPythonTest
     EXPRESSION "instance = itk.VTKPolyDataMeshIO.New()"
   )

--- a/Modules/Filtering/MeshToPolyData/wrapping/test/CMakeLists.txt
+++ b/Modules/Filtering/MeshToPolyData/wrapping/test/CMakeLists.txt
@@ -23,7 +23,7 @@ execute_process(
 if(_have_numpy_return_code EQUAL 0)
   itk_python_add_test(NAME itkPolyLineCellTest COMMAND itkPolyLineCellTest.py)
   itk_python_add_test(
-    NAME itkPyVectorContainerTest
+    NAME itkMeshToPolyDataPyVectorContainerTest
     COMMAND
       ${CMAKE_CURRENT_SOURCE_DIR}/itkPyVectorContainerTest.py
   )

